changelogs/fragments/2334-redfish_config-skip-incorrect-attributes.yml

```
@@ -0,0 +1,4 @@
+minor_changes:
+  - plugins/module_utils/redfish_utils.py - Modified set_bios_attributes function to skip invalid attribute instead of returning. Added skipped attributes to output (https://github.com/ansible-collections/community.general/issues/1995)
+  - plugins/modules/remote_management/redfish/idrac_redfish_config.py - Modified set_manager_attributes function to skip invalid attribute instead of returning. Added skipped attributes to output. Modified module exit to add warning variable (https://github.com/ansible-collections/community.general/issues/1995)
+  - plugins/modules/remote_management/redfish/redfish_config.py Modified module exit to add warning variable (https://github.com/ansible-collections/community.general/issues/1995)
```

```suggestion
  - redfish_utils module utils - modified set_bios_attributes function to skip invalid attribute instead of returning. Added skipped attributes to output (https://github.com/ansible-collections/community.general/issues/1995).
  - idrac_redfish_config - modified set_manager_attributes function to skip invalid attribute instead of returning. Added skipped attributes to output. Modified module exit to add warning variable (https://github.com/ansible-collections/community.general/issues/1995).
  - redfish_config - modified module exit to add warning variable (https://github.com/ansible-collections/community.general/issues/1995).
```
I would suggest to further rewrite the changelog fragments so that they actually help users who don't know the code of the modules.

---

Fixed.